### DISCUSSION TITLE
What does "not stable" mean in the context of `LibSort`?

### DISCUSSION THREAD

**Discussion by PaulRBerg** [Vectorized/solady#304] [Q&A] [answered]
Thanks for building this amazing Solidity library, @Vectorized!

I was looking at the `LibSort` library, and I saw this comment:

https://github.com/Vectorized/solady/blob/7143fa2d068537c8a2692c9612f7a1c67cb0e185/src/utils/LibSort.sol#L64

Would you mind explaining what does "not stable" mean in this context?

**Comment by Vectorized** [comment]
I think the "NOT stable" comment here can be removed, since the sort operates on uint256s. 

Sort stability only matters when sorting tuples.

**Reply by PaulRBerg** [reply, marked as answer]
I see, thanks for explaining. It looks like you removed the comments in the meantime.

https://github.com/Vectorized/solady/commit/ed18bb65a0e72656a2f64256dfc4d78eccb33a1a

**Comment by Dosulute-chemist** [comment]
It would be interesting to have more standardized gas benchmarks for the optimized snippets, especially comparing them against equivalent implementations using Solidity’s newer compiler optimizations. This could help developers understand the actual gas savings and the trade-offs in readability when choosing Solady
